pymongo/encryption.py

```diff
@@ -83,9 +83,8 @@
 _KMS_CONNECT_TIMEOUT = CONNECT_TIMEOUT  # CDRIVER-3262 redefined this value to CONNECT_TIMEOUT
 _MONGOCRYPTD_TIMEOUT_MS = 10000
 
-
-_DATA_KEY_OPTS: CodecOptions[SON[str, Any]] = CodecOptions(
-    document_class=SON[str, Any], uuid_representation=STANDARD
+_DATA_KEY_OPTS: CodecOptions[dict[str, Any]] = CodecOptions(
```

Comment:
This `Dict` is a runtime object, that's what the annotations import doesn't help.

Reply:
Ahhh right, we're passing it directly as a keyword argument.